ui/src/components/VolumeMetricGraphCard.js

```diff
@@ -161,20 +156,26 @@ const MetricGraphCard = (props) => {
 
   // slot[0] => timestamp
   // slot[1] => value
-  const volumeUsageData = volumeUsedOperated?.map((slot) => {
+  const volumeUsageData = volumeUsageOperated?.map((slot) => {
     return {
       date: new Date(slot[0] * 1000), // convert from the RFC 3339 to time in JS
-      y:
-        slot[1] === null
-          ? null
-          : Math.round((slot[1] / volumeStorageCapacity) * 100),
+      y: slot[1] === null ? null : Math.round(slot[1] * 100),
     };
   });
 
-  const volumeLatencyData = volumeLatencyOperated?.map((slot) => {
+  const volumeLatencyWriteData = volumeLatencyWriteOperated?.map((slot) => {
     return {
       date: new Date(slot[0] * 1000),
-      y: slot[1] === null ? null : Math.round(slot[1] * 1000000),
+      write: slot[1] === null ? null : Math.round(slot[1] * 1000000),
```

Comment:
ditto, here: instead of having a mysterious `1000000` value used for multiplication, we could hide the computation in a function `toGb` (or a better name :stuck_out_tongue: ) and reuse this function where it's needed.

Reply:
Thanks for the idea! 
It's a more understandable code!